_submissions/example.md

```diff
@@ -0,0 +1,30 @@
+# Module & Lua Script info
```

Comment:
not sure if everything under `_submission` is still needed as we read from GitHub directly. @Helias